bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -0,0 +1,660 @@
+            },
+            {
+                name: "users",
+                columns: [
+                    { name: "created_date_time", type: "DATETIME" },
+                    { name: "email", type: "STRING", length: 255 },
+                    { name: "first_name", type: "STRING", length: 255 },
+                    { name: "job_title", type: "STRING", length: 255 },
+                    { name: "last_name", type: "STRING", length: 255 },
+                    { name: "user_id", type: "STRING", length: 255, primaryKey: true },
+                    { name: "user_name", type: "STRING", length: 255 },
+                    { name: "user_status", type: "STRING", length: 255 },
+                    { name: "user_type", type: "STRING", length: 255 },
+                ]
+            }
+        ],
+        relationships: [
+            {
+                name: "nested_table_1",
+                primaryTable: "templates",
+                foreignTable: "templates_documents",
+                columnPairs: [
+                    {
+                        primaryKey: "template_id",
+                        foreignKey: "parent_template_id"
+                    }
+                ]
+            },
+            {
+                name: "nested_table_2",
+                primaryTable: "templates",
+                foreignTable: "templates_folder_ids",
+                columnPairs: [
+                    {
+                        primaryKey: "template_id",
+                        foreignKey: "parent_template_id"
+                    }
+                ]
+            },
+            {
+                name: "nested_table_3",
+                primaryTable: "envelopes",
+                foreignTable: "envelopes_recipients_signers",
+                columnPairs: [
+                    {
+                        primaryKey: "envelope_id",
+                        foreignKey: "parent_envelope_id"
+                    }
+                ]
+            }
+        ]
+    },
+    actions: [
+        {
+            name: "Add_Recipient",
+            parameters: [
+                { name: "envelope_Id", type: "STRING" },
+                { name: "recipient_Name", type: "STRING" },
+                { name: "recipient_Email", type: "STRING" },
+                { name: "role_Name", type: "STRING" },
+                { name: "recipient_Email_Unknown", type: "STRING" },
+                { name: "user_Id", type: "STRING" },
+            ],
+            function: addRecipient
+        },
+        {
+            name: "Resend_Envelope",
+            parameters: [
+                { name: "envelope_Id", type: "STRING" },
+                { name: "recipient_Name", type: "STRING" },
+                { name: "recipient_Id", type: "STRING" },
+                { name: "recipient_Email", type: "STRING" },
+                { name: "user_Id", type: "STRING" },
+            ],
+            function: resendEnvelope
+        },
+        {
+            name: "Send_Envelope",
```

Comment:
As you have now just one service action, you can have just one button in microapp and set only the first recipient required and rest optional.

Reply:
Changes implemented